agent-bootstrap/src/main/java/io/opentelemetry/auto/typedspan/DbCassandraSpan.java

```diff
@@ -0,0 +1,563 @@
+   *     <p>The value may be sanitized to exclude sensitive information.
+   */
+  @Override
+  public DbCassandraSemanticConvention setDbStatement(String dbStatement) {
+    status.set(AttributeStatus.DB_STATEMENT);
+    delegate.setAttribute("db.statement", dbStatement);
+    return this;
+  }
+
+  /**
+   * Sets db.operation.
+   *
+   * @param dbOperation The name of the operation being executed, e.g. the [MongoDB command
+   *     name](https://docs.mongodb.com/manual/reference/command/#database-operations) such as
+   *     `findAndModify`..
+   *     <p>While it would semantically make sense to set this, e.g., to a SQL keyword like `SELECT`
+   *     or `INSERT`, it is not recommended to attempt any client-side parsing of `db.statement`
+   *     just to get this property (the back end can do that if required).
+   */
+  @Override
+  public DbCassandraSemanticConvention setDbOperation(String dbOperation) {
+    status.set(AttributeStatus.DB_OPERATION);
+    delegate.setAttribute("db.operation", dbOperation);
+    return this;
+  }
+
+  /**
+   * Sets net.peer.name.
+   *
+   * @param netPeerName Remote hostname or similar, see note below..
+   */
+  @Override
+  public DbCassandraSemanticConvention setNetPeerName(String netPeerName) {
+    status.set(AttributeStatus.NET_PEER_NAME);
+    delegate.setAttribute("net.peer.name", netPeerName);
+    return this;
+  }
+
+  /**
+   * Sets net.peer.ip.
+   *
+   * @param netPeerIp Remote address of the peer (dotted decimal for IPv4 or
+   *     [RFC5952](https://tools.ietf.org/html/rfc5952) for IPv6).
+   */
+  @Override
+  public DbCassandraSemanticConvention setNetPeerIp(String netPeerIp) {
+    status.set(AttributeStatus.NET_PEER_IP);
+    delegate.setAttribute("net.peer.ip", netPeerIp);
+    return this;
+  }
+
+  /**
+   * Sets net.peer.port.
+   *
+   * @param netPeerPort Remote port number..
+   */
+  @Override
+  public DbCassandraSemanticConvention setNetPeerPort(long netPeerPort) {
+    status.set(AttributeStatus.NET_PEER_PORT);
+    delegate.setAttribute("net.peer.port", netPeerPort);
+    return this;
+  }
+
+  /**
+   * Sets net.transport.
+   *
+   * @param netTransport Transport protocol used. See note below..
+   */
+  @Override
+  public DbCassandraSemanticConvention setNetTransport(String netTransport) {
+    status.set(AttributeStatus.NET_TRANSPORT);
+    delegate.setAttribute("net.transport", netTransport);
+    return this;
+  }
+
+  /**
+   * Sets db.cassandra.keyspace.
+   *
+   * @param dbCassandraKeyspace The name of the keyspace being accessed. To be used instead of the
+   *     generic `db.name` attribute..
+   */
+  @Override
+  public DbCassandraSemanticConvention setDbCassandraKeyspace(String dbCassandraKeyspace) {
+    status.set(AttributeStatus.DB_CASSANDRA_KEYSPACE);
+    delegate.setAttribute("db.cassandra.keyspace", dbCassandraKeyspace);
+    return this;
+  }
+
+  /** Builder class for {@link DbCassandraSpan}. */
+  public static class DbCassandraSpanBuilder {
```

Comment:
I still want for this to extend `Span.Builder` :( To allow using `SemanticAttribute.set`. And generic `setAttribute` as well

Reply:
I'll mark this conversation resolved since we agree that the method at https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/502/files#diff-55139a95666573cc2dd596464df107a5R346 suffices